pkg/git/git.go

```diff
@@ -292,7 +295,7 @@ func configSparseCheckout(logger *zap.SugaredLogger, spec FetchSpec) error {
 			logger.Errorf("failed to get current directory: %v", err)
 			return err
 		}
-		file, err := os.OpenFile(filepath.Join(cwd, ".git/info/sparse-checkout"), os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+		file, err := os.OpenFile(filepath.Join(cwd, ".git/info/sparse-checkout"), os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0o644)
```

Comment:
why do we want to change this? Seems not relevant. Correct me if I'm wrong.

Reply:
The vscode format it automaticly, I will try to recovery it. The `o` represent the number is a `eight number`. It is as same as before, just for user read easily. The `b` represent the number is a `binary number`...

Reply:
Thank you!